docs/yaml.rst

```diff
@@ -134,13 +133,11 @@ It has the following sub-sections:
 Encoder Description
 ~~~~~~~~~~~~~~~~
 
-The **ioda** section defines the ObsGroup objects that will be created. Here is an example:
+The **encoder** section defines the ObsGroup objects that will be created. Here is an example:
 
 .. code-block:: yaml
 
   encoder:
```

Comment:
Do you not specify in the YAML which encoder you want to use?

Reply:
Same question.  
```
encoder:
  type: netcdf
  dimensions:
  globals:
  variables:
```

Currently the `type` does not have any effect for netcdf or ioda format.  
Do we need to specify type to zarr when the output is zarr?

Reply:
The description for the encoder is currently not dependent on the encoder type. This is partially due to the fact that all the output formats are similar to HDF5. Currently no need to indicate the output type in the YAML file (the spec is the same).

There is no ability to output BUFR files of any kind... This would be a HUGE deal, probably leading us to dump the dependency on NCEPLib-bufr.

The IODA Encoder for bufr is part of the IODA project, and is not documented here...

Reply:
@emilyhcliu The **type** attribute is not used or needed at all.

Reply:
ok thanks @rmclaren for the clarification